src/vector_commitment/vector_commitment.cairo

```diff
@@ -157,9 +159,12 @@ fn shift_queries(
 
 fn hash_blake_or_poseidon(x: felt252, y: felt252, is_verifier_friendly: bool) -> felt252 {
```

Comment:
Since the hash function was changed, we should also update function name to `hash_blake_or_pedersen`.

Reply:
resolved in this commit https://github.com/HerodotusDev/cairo-verifier/pull/76/commits/2f0fc0b486d37e767dbb385b5e5f80c35eecef2f

Reply:
Ok, would be better in this PR, but we can leave it like this